DASHBOARD_README.md

```diff
@@ -0,0 +1,231 @@
+# GR Incremental Trace Viewer Dashboard
```

Comment:
This feels like more like a README than overly dashboard specific. I would maybe more replace our readme with this.
Just ensure nothing relevant is lost of our current readme.